src/main/java/com/google/json/JsonSanitizer.java

```diff
@@ -513,6 +513,7 @@ private void sanitizeString(int start, int end) {
     for (int i = start; i < end; ++i) {
       char ch = jsonish.charAt(i);
       switch (ch) {
+        case '\t': replace(i, i + 1, "\\t"); break; // tabs needs to be escaped, also
```

Comment:
The "tabs need to be escaped, also" is a bit confusing outside the context of this PR.

Could the comment mention that ECMA-404 requires escaping for all control characters?